piccolo_api/fastapi/endpoints.py

```diff
@@ -506,6 +506,37 @@ def modify_signature(
                     ]
                 )
 
+            parameters.extend(
+                [
+                    Parameter(
+                        name="__range_header",
+                        kind=Parameter.POSITIONAL_OR_KEYWORD,
+                        annotation=bool,
+                        default=Query(
+                            default=False,
+                            description=(
+                                "enable to add range-header respons header"
```

Comment:
For the description, how about:

`Set to 'true' to add the Content-Range response header.`